model/clusters_mgmt/v1/cluster_type.model

```diff
@@ -225,5 +225,11 @@ class Cluster {
 	DeleteProtection DeleteProtection
 
 	// Details for `htpasswd` identity provider.
-	Htpasswd HTPasswdIdentityProvider	
+	Htpasswd HTPasswdIdentityProvider
+
+	// ID of private hosted zone
```

Comment:
It needs the ending '.' to include it in the openapi (swagger) page

Reply:
TIL

Reply:
@gdbranco 
thanks for letting me know this
I already added the ending ".". Please help to review again